### DISCUSSION TITLE
DietPi-Software | Komodo

### DISCUSSION THREAD

**Discussion by aeonoea** [MichaIng/DietPi#7333] [Software Request]
Komodo is a docker container manager, arguably more intuitive and more beautiful (and more open-source) than Portainer.

https://github.com/mbecker20/komodo

> There is no limit to the number of servers you can connect, and there will never be. There is no limit to what API you can use for automation, and there never will be. No "business edition" here.

I'd like to see it as a docker container manager in dietpi-software and surprised that this has not been posted here yet.

[Demo](https://demo.komo.do)
(login: demo / pw: demo)

**Comment by webfinder42** [comment]
1+ for this one.
More info here: https://komo.do
